README.md

```diff
@@ -253,11 +256,12 @@ and then build and run the image as follows:
 
     docker build . -t allinurl/goaccess
     docker run --restart=always -d -p 7890:7890 \
-      -v "/srv/goaccess/data:/srv/data"         \
-      -v "/srv/goaccess/html:/srv/report"       \
-      -v "/var/log/apache2:/srv/logs"           \
-      --name=goaccess allinurl/goaccess
-
+      -v "/srv/goaccess/data:/goaccess" \
+      -v "/srv/goaccess/html/index.html:/goaccess/index.html" \
+      -v "/var/log/apache2/access.log:/goaccess/access.log" \
+      --name=goaccess allinurl/goaccess \
```

Comment:
This part of the readme used to instruct to run the docker container without specifying any command. For me, this is no longer working, I have to specify a command.

So this part is now a bit redondant with the below part talking about command that can be passed to customize behavior.